dev-python/pyperclip/pyperclip-1.9.0.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{10..13} )
+
+inherit distutils-r1 virtualx pypi
+
+DESCRIPTION="A cross-platform clipboard module for Python"
+HOMEPAGE="
+	https://github.com/asweigart/pyperclip/
+	https://pypi.org/project/pyperclip/
+"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~mips ~ppc64 ~riscv ~s390 ~sparc ~x86"
+
+PATCHES=(
+	"${FILESDIR}/${P}-fix-test.patch"
+)
```

Comment:
Oh, and it goes into `src_prepare` if it's defined. But I'll fix that and merge, don't worry.